### PR TITLE
Assign to Team

### LINKED ISSUE #4691: Assign to Team after reaching lvl 5
### Short Description

Add the ability to assign to a team (blue, red, yellow) after bot is reaching lvl5

### PR DESCRIPTION
**Pull request by DeXtroTip**
## Short Description:
It makes it able to choose a team when reaching level 5.
Not sure if it is still wanted since some may think it is better done manually.
As someone said before, if it doesn't feel right having this inside CompleteTutorial task, I can create a new task for this.

Haven't updated docs file since there were nothing about this task there, will do it later on in another PR.

## Fixes/Resolves/Closes (please use correct syntax):
- Closes #4691

### CODE REVIEW

File: pokemongo_bot/cell_workers/complete_tutorial.py [hunk 1]
```diff
@@ -15,23 +16,22 @@ def initialize(self):
         self.api = self.bot.api
         self.nickname = self.config.get('nickname','')
         self.team = self.config.get('team',0)
-        self.may_run = True
-
-    def should_run(self):
-        return self.may_run
+        self.tutorial_run = True
+        self.team_run == True
```
Comment (sohje) [inline pokemongo_bot/cell_workers/complete_tutorial.py]: self.team_run = True?

File: pokemongo_bot/cell_workers/complete_tutorial.py [hunk 2]
```diff
@@ -179,3 +179,36 @@ def _set_tutorial_state(self, completed):
         except KeyError:
             self.logger.error("KeyError while setting tutorial state")
             return False
+
+    def _set_team(self):
+        if self.team == 0:
```
Comment (sohje) [inline pokemongo_bot/cell_workers/complete_tutorial.py]: Purpose? And where did you set it before?

Comment (DeXtroTip) [inline pokemongo_bot/cell_workers/complete_tutorial.py]: At `initialize`:
`self.team = self.config.get('team',0)`
If it is 0 then no team must be picked.

Comment (sohje) [inline pokemongo_bot/cell_workers/complete_tutorial.py]: Sorry, didnt see that 😆

File: configs/config.json.cluster.example [hunk 3]
```diff
@@ -26,7 +26,9 @@
         "config": {
           "enabled": false,
           "// set a name": "",
-          "nickname": ""
+          "nickname": "",
+          "// 0 = No Team, 1 = Blue, 2 = Red, 3 = Yellow": "",
```
Comment (solderzzc) [inline configs/config.json.cluster.example]: Red is 2 ... I thought Red is 3...

Comment (sohje) [inline configs/config.json.cluster.example]:
```python
syntax = "proto3";
package POGOProtos.Enums;

enum TeamColor {
	NEUTRAL = 0;
	BLUE = 1;
	RED = 2;
	YELLOW = 3;
}
```

Comment (solderzzc) [inline configs/config.json.cluster.example]: Yes, that's wired, the red always jumped as the 3rd one.

**Comment by solderzzc** [comment]
👍

**Comment by k4n30** [comment]
Uhhh ..... did the automerge not work?

**Comment by solderzzc** [comment]
👍

**Comment by k4n30** [comment]
@solderzzc think it's broken :(

**Comment by solderzzc** [comment]
👍

**Comment by solderzzc** [comment]
@k4n30 let me try it..

**Comment by solderzzc** [comment]
Oh, @DeXtroTip  already fixed it.

**Pull request merged**